### DISCUSSION TITLE
Example of configuring a HA user interface for the live/event streams?

### DISCUSSION THREAD

**Discussion by cerebrate** [tsightler/ring-mqtt#372] [Support Q&A] [answered]
Before I set out to reinvent the wheel, are there any good examples out there?

(Side note: this would also be a great thing to have on the wiki, here.)

**Comment by tsightler** [comment, marked as answer]
I posted my personal config on the HA forum some weekds ago, although it's really nothing special, just a picture glance cards with the entities and cameras I personally prefer.

https://community.home-assistant.io/t/ring-device-integration-via-mqtt-w-video-streaming/102105/831?u=tsightler

**Reply by cerebrate** [reply]
Thanks! That was a lot simpler than I was expecting it to be, and it works great for me.
